docs/content/integrations/embedded-elt.mdx

```diff
@@ -135,12 +134,12 @@ replication_config = file_relative_path(__file__, "../sling_replication.yaml")
 sling_resource = SlingResource(connections=[...])  # Add connections here
 
 
-@sling_assets(replication_config=replication_config)
+@sling_assets(
+    replication_config=replication_config,
+    dagster_sling_translator=DagsterSlingTranslator(),
```

Comment:
If it's not a custom one, it shouldn't be provided in the examples for ✨elegance✨